core/templates/pages/skill-editor-page/modal-templates/add-worked-example-modal.controller.ts

```diff
@@ -36,6 +36,13 @@ angular.module('oppia').controller('AddWorkedExampleModalController', [
     $scope.tmpWorkedExampleQuestionHtml = '';
     $scope.tmpWorkedExampleExplanationHtml = '';
 
+    $scope.isEditorContentLengthExceeded = function() {
+      return (
+        $scope.tmpWorkedExampleQuestionHtml.length > 1000 ||
+        $scope.tmpWorkedExampleExplanationHtml.length > 1000
+      );
+    };
```

Comment:
I think it's a good idea to break these 2 checks into 2 different functions (and 2 different errors) as if they're completely separate! This is good because: a lot of people will see these errors as soon as we merge this PR, and it may be confusing whether the Question or the Explanation (or both) have errors.

Reply:
Sure , i will resolve that asap .